db/db_impl.cc

```diff
@@ -4000,18 +4000,30 @@ Status DBImpl::GetImpl(const ReadOptions& read_options,
   auto cfh = reinterpret_cast<ColumnFamilyHandleImpl*>(column_family);
   auto cfd = cfh->cfd();
 
+  // Acquire SuperVersion
+  SuperVersion* sv = GetAndRefSuperVersion(cfd);
+
+  TEST_SYNC_POINT("DBImpl::GetImpl:1");
```

Comment:
I am a bit confused why did we move the sync point here ?
Should not the sync_point be after the call to LastSequence() ?
So that in the test we issue a Put+Flush after we call LastSequence()